ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -855,6 +1087,44 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '6A2C', source: 'ヘスペロス', capture: false }),
       response: Responses.bigAoe(),
     },
+    {
+      id: 'P4S Act Three Bait Order',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '00AD', source: 'Hesperos' }),
+      netRegexDe: NetRegexes.tether({ id: '00AD', source: 'Hesperos' }),
+      netRegexFr: NetRegexes.tether({ id: '00AD', source: 'Hespéros' }),
+      netRegexJa: NetRegexes.tether({ id: '00AD', source: 'ヘスペロス' }),
+      condition: (data) => data.act === '3',
+      // Tethers come out East or West (0 seconds), (3s) Middle knockack, (6) Opposite Cardinal
+      suppressSeconds: 7,
+      infoText: (data, matches, output) => {
+        const thorn = (data.thornIds ??= []).indexOf(parseInt(matches.sourceId, 16));
+        const num = parseInt(matches.sourceId, 16);
+        console.error(`Act Two: Found thorn index of ${thorn} and id ${num} with ${matches.sourceId}`);
```

Comment:
Should this only print out if there's an error?

Reply:
Woops, left that one in from copy paste.